src/services/routePathLinkService.ts

```diff
@@ -5,20 +5,37 @@ import notificationStore from '~/stores/notificationStore';
 import NotificationType from '~/enums/notificationType';
 import RoutePathLinkFactory from '~/factories/routePathLinkFactory';
 import IExternalLink from '~/models/externals/IExternalLink';
+import { AddLinkDirection } from '~/stores/routePathStore';
 import GraphqlQueries from './graphqlQueries';
 
 // TODO: create two services, RoutePathLinkService and LinkService?
 class RoutePathLinkService {
     public static
-        async fetchAndCreateRoutePathLinksWithStartNodeId(nodeId: string, orderNumber: number)
+        async fetchAndCreateRoutePathLinksWithNodeId
+        (nodeId: string, linkDirection: AddLinkDirection, orderNumber: number)
```

Comment:
Should orderNumber atleast be optional to make this more reusable?

Reply:
But that again will require us to implement logic, that can be used to create invalid routePathLinks. RoutePathLinks are only valid if they have orderNumber.